cmd/queue.go

```diff
@@ -0,0 +1,113 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/strangelove-ventures/heighliner/builder"
+)
+
+type GithubRelease struct {
+	TagName string `json:"tag_name"`
+}
+
+func mostRecentReleasesForChain(
+	chainNodeConfig builder.ChainNodeConfig,
+	number int16,
+) (builder.HeighlinerQueuedChainBuilds, error) {
+	if chainNodeConfig.GithubOrganization == "" || chainNodeConfig.GithubRepo == "" {
+		return builder.HeighlinerQueuedChainBuilds{}, fmt.Errorf("github organization: %s and/or repo: %s not provided for chain: %s\n", chainNodeConfig.GithubOrganization, chainNodeConfig.GithubRepo, chainNodeConfig.Name)
+	}
+	client := http.Client{Timeout: 5 * time.Second}
+
+	req, err := http.NewRequest(http.MethodGet, fmt.Sprintf("https://api.github.com/repos/%s/%s/releases?per_page=%d&page=1",
+		chainNodeConfig.GithubOrganization, chainNodeConfig.GithubRepo, number), http.NoBody)
+	if err != nil {
+		return builder.HeighlinerQueuedChainBuilds{}, fmt.Errorf("error building github releases request: %v", err)
+	}
+
+	basicAuthUser := os.Getenv("GITHUB_USER")
+	basicAuthPassword := os.Getenv("GITHUB_PASSWORD")
```

Comment:
Should this be surfaced in documentation somewhere?

Reply:
These were copy and pasted. I don't think they are used for anything though. I'd rather remove them than document them when they aren't tested.